src/itemlistformaction.cpp

```diff
@@ -1538,12 +1534,12 @@ void ItemListFormAction::handle_op_saveall()
 
 	bool overwrite_all = false;
 	for (size_t item_idx = 0; item_idx < filenames.size(); ++item_idx) {
-		const auto filename = filenames[item_idx];
-		const auto filepath = directory + filename;
+		const auto filename = filenames[item_idx].clone();
+		const auto filepath = directory.join(filename).clone();
```

Comment:
Same question: why do we need `clone()` here?

Reply:
Line 1537 not compiled without clone. Line 1538 I fixed.